### PR TITLE
Rename 'variable' to 'name' in source code and unit tests

### LINKED ISSUE #20: Rename "variable" to "name"
Currently in the [exported json](https://raw.githubusercontent.com/plentycode/sass-export/develop/exported-examples/annotations.json) the variable name is called `variable` please rename it to "name"

```json
{
  "fonts": [
        { "variable": "$font-size", "value": "16px", "compiledValue": "16px" },
        { "variable": "$font-color", "value": "$brand-gray-medium", "compiledValue":"#d6d6d6" }
    ]
}
```

We want it to be:
```json
{
  "fonts": [
        { "name": "$font-size", "value": "16px", "compiledValue": "16px" },
        { "name": "$font-color", "value": "$brand-gray-medium", "compiledValue":"#d6d6d6" }
    ]
}
```

Don't forget to update:
- [ ] source code: #31
- [ ] unit tests: #31
- [x] documentation:  #29
- [x] examples: #22, #24, #25, #26, #27, #28

### PR DESCRIPTION
**Pull request by puneet97**
Fixes #20

### CODE REVIEW

File: src/app/converter/converter.test.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe('Converter class', () => {
   });
 
 
-  describe('JSON exported extructure', () => {
+  describe('JSON exported structure', () => {
```
Comment (civan) [inline src/app/converter/converter.test.ts]: great catch , thanks

**Comment by civan** [comment]
awesome, thanks!

**Comment by puneet97** [comment]
@civan You might want to consider closing #31 now that #20 is completed/closed.

**Comment by civan** [comment]
@puneet97 done, thanks

**Pull request merged**